gulpfile.js

```diff
@@ -267,7 +267,7 @@ gulp.task('copy', 'Copy site files (polyfills, templates, etc.) to dist/', funct
   let app = gulp.src([
       '*',
       'app/manifest.json',
-      '!{README.md,package.json,gulpfile.js,test_runner.py}',
+      '!{README.md,package.json,gulpfile.js}',
```

Comment:
why are we putting the `test_runner` in the `dist`, is that where the tests get run?